src/seedu/addressbook/data/person/Person.java

```diff
@@ -34,14 +36,17 @@ public Person(Name name, Phone phone, Email email, Address address, Set<Tag> tag
      * Copy constructor.
      */
     public Person(ReadOnlyPerson source) {
-        this(source.getName(), source.getPhone(), source.getEmail(), source.getAddress(), source.getTags());
+        this(source.getName(), source.getGender(), source.getPhone(), source.getEmail(), source.getAddress(), source.getTags());
     }
 
     @Override
     public Name getName() {
         return name;
     }
 
+    @Override
+    public Gender getGender() { return gender; }
```

Comment:
Probably would be better to stick to the K&R style of brackets as seen in the adjacent methods.